core/src/test/java/org/ehcache/core/UserManagedCacheTest.java

```diff
@@ -100,7 +94,6 @@ private void assertFailingTransitionGoesToLowestStatus(InternalCache cache) thro
     } catch (StateTransitionException e) {
       assertThat(cache.getStatus(), is(Status.UNINITIALIZED));
     }
-
```

Comment:
Yes, this could be reverted.  I had originally touched the file because of some test that was subsequently relocated (I'd put it in the wrong place).